src/test/java/com/segment/analytics/android/integrations/comscore/ComScoreTest.java

```diff
@@ -553,73 +522,63 @@ public void videoContentStartedWithTVAirdate() {
     expected.put("c4", "*null");
     expected.put("c6", "*null");
 
-    Mockito.verify(streamingAnalytics).notifyPlay(70);
-    Mockito.verify(playbackSession).setAsset(expected);
+    Mockito.verify(streamingAnalytics).startFromPosition(70);
+    Mockito.verify(streamingAnalytics).notifyPlay();
+    Mockito.verify(streamingAnalytics,atLeast(2))
```

Comment:
Same here, do we expect to set metadata twice?

Reply:
`Mockito.verify(streamingAnalytics,atLeast(1)).setMetadata(refEq(getContentMetadata(expected)));`